iglu-registry/README.md

```diff
@@ -0,0 +1,205 @@
+# Iglu Registry
+
+## Introduction
+
+This folder contains
+
+* docker images of [Iglu Registry](iglu-registry)
+  * `0.3.0`
+* a docker compose example to run [Iglu Registry](iglu-registry) with [Postgres](https://github.com/docker-library/postgres)
+
+[Iglu Registry](iglu-registry) images are based on [Snowplow base image][base-image] which leverages [the Java 8 Alpine image][alpine-image].
+
+The S3 Loader runs under [dumb-init][dumb-init] which handles reaping zombie processes
```

Comment:
s3 loader

Reply:
Ah, copy-pasting common stuff, will fix it.